dask/dataframe/core.py

```diff
@@ -2300,15 +2301,24 @@ def idxmin(self, axis=None, skipna=True, split_every=False):
                 result.divisions = (min(self.columns), max(self.columns))
             return result
 
-    @_numeric_only
     @derived_from(pd.DataFrame)
-    def count(self, axis=None, split_every=False, numeric_only=None):
+    def count(self, axis=None, split_every=False, numeric_only=False):
+        # This method is shared by DataFrame / Series, but only DataFrame
```

Comment:
Perhaps we should actually split the two methods, now that there is a difference in signature?

Reply:
Series actually has it' own method and calls this one.